.github/workflows/main.yaml

```diff
@@ -59,19 +59,3 @@ jobs:
       track-name: 1.4
     secrets:
       CHARMCRAFT_AUTH: ${{ secrets.CHARMCRAFT_AUTH }}
-
-  publish-charm-on-push:
-    name: Publish Developer Charm To Branch
-    needs:
-      - lint-report
-      - static-analysis
-      - unit-tests-with-coverage
-      - integration-test
-    if: ${{ (github.ref_name != 'v1.4') && (github.event_name == 'push') }}
-    uses: canonical/sdcore-github-workflows/.github/workflows/publish-charm.yaml@v0.0.1
-    with:
-      branch-name: ${{ github.ref_name }}
-      charm-file-name: "sdcore-amf-k8s_ubuntu-22.04-amd64.charm"
-      track-name: 1.4
-    secrets:
-      CHARMCRAFT_AUTH: ${{ secrets.CHARMCRAFT_AUTH }}
```

Comment:
When squashing, please make sure the context for this change is included in the commit message, as otherwise the history will only show the title of this PR.